file2.py

```diff
@@ -1,2 +1,7 @@
+import math
+#printing x before decalring 
+print(x) 
```

Comment:
printing x before declaration
```suggestion
x = 20
print(x) 
```